src/gpu/d3d12/SDL_gpu_d3d12.c

```diff
@@ -1952,6 +1935,204 @@ static void D3D12_INTERNAL_TrackComputePipeline(
 
 #undef TRACK_RESOURCE
 
+
+// Debug Naming
+
+static bool D3D12_INTERNAL_StrToWStr(
```

Comment:
Is there a reason you aren't using WIN_UTF8ToStringW()?

Reply:
The reason is that I didn't know it existed!